doc/extensions/zephyr/application.py

```diff
@@ -48,6 +48,9 @@ class ZephyrAppCommandsDirective(Directive):
     \:board:
       if set, the application build will target the given board.
 
+    \:shield:
```

Comment:
Please update the documentation in ``doc/documentation/doc-guidelines.rst`` with this new argument too.

Reply:
... actually I'm making edits to the doc-guidelines.rst file right now, so I'll just add it while I'm there.